test/cmd.js

```diff
@@ -895,6 +895,60 @@ describe('express(1)', function () {
         .expect(404, /<h1>Not Found<\/h1>/, done)
       })
     })
+
+    describe('--view=none', function () {
```

Comment:
I believe this is already in the `--view <engine>` section, and renaming it would just be reverting would be almost the opposite of 83858da2

I can do that, but so far as I can tell it's just a naming thing originally `--none` -> `--view=none` -> `none`